app/lib/admin/backend.dart

```diff
@@ -312,6 +324,48 @@ class AdminBackend {
             .toList())
         .toSet();
 
+    final pool = Pool(10);
+    final futures = <Future>[];
+    versions.forEach((final v) {
+      futures.add(pool.withResource(
+          () => packageBackend.removePackageTarball(packageName, v)));
```

Comment:
will a 404 error be ignored?

Reply:
Yes, they are ignored. Also added a comment about it here.